kafka-connector/README.md

```diff
@@ -89,8 +89,11 @@ Connector supports the following configs:
 | kafka.key.attribute | String | null | The Cloud Pub/Sub message attribute to use as a key for messages published to Kafka. |
 | kafka.partition.count | Integer | 1 | The number of Kafka partitions for the Kafka topic in which messages will be published to. NOTE: this parameter is ignored if partition scheme is "kafka_partitioner".|
 | kafka.partition.scheme | round_robin, hash_key, hash_value, kafka_partitioner | round_robin | The scheme for assigning a message to a partition in Kafka. The scheme "round_robin" assigns partitions in a round robin fashion, while the schemes "hash_key" and "hash_value" find the partition by hashing the message key and message value respectively. "kafka_partitioner" scheme delegates partitioning logic to kafka producer, which by default detects number of partitions automatically and performs either murmur hash based partition mapping or round robin depending on whether message key is provided or not.|
-| gcp.credentials.file.path | String | Optional | The file path, which stores GCP credentials.| If not defined, GOOGLE_APPLICATION_CREDENTIALS env is used. |
-| gcp.credentials.json | String | Optional | GCP credentials JSON blob | If specified, use the explicitly handed credentials. Consider using the externalized secrets feature in Kafka Connect for passing the value. |
+| gcp.credentials.file.path | String | Optional | The file path, which stores GCP credentials. If not defined, GOOGLE_APPLICATION_CREDENTIALS env is used. |
+| gcp.credentials.json | String | Optional | GCP credentials JSON blob. If specified, use the explicitly handed credentials. Consider using the externalized secrets feature in Kafka Connect for passing the value. |
+| max.ack.batch.size | Integer | 1000 | The maximum number of ack ids to batch per ack request to Cloud Pub/Sub. |
```

Comment:
It is payload size limit that is being hit here: 

> Request payload size exceeds the limit: 524288 bytes.

This limit has been changing in the past (https://github.com/googleapis/google-cloud-go/issues/452), so it is handy to have control over it especially since users do build this connector themselves and might choose different versions of google cloud client.

Speaking of limit specifics: it is simpler to set it in number of messages as opposed to approach taken in the link above.

Reply:
We could just set this to a conservative number like 100 or 200 and use that. Unless someone were to alter the pom.xml, they are not going to use their own version of the cloud client and in fact I would recommend against people changing the version of the client pulled into the connector because there would be no guarantee of compatibility without having to make code changes. I just don't think this is something that needs to be configurable by users.